test/e2e/curated_packages_test.go

```diff
@@ -4,32 +4,172 @@
 package e2e
 
 import (
+	"context"
 	"fmt"
+	"strconv"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/aws/eks-anywhere/internal/pkg/api"
 	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
 	"github.com/aws/eks-anywhere/test/framework"
 )
 
 const (
-	eksaPackageControllerHelmChartName = "eks-anywhere-packages"
-	eksaPackageControllerHelmURI       = "oci://public.ecr.aws/eks-anywhere/eks-anywhere-packages"
-	eksaPackageControllerHelmVersion   = "0.1.10-eks-a-10"
-	eksaPackageBundleBaseURI           = "oci://public.ecr.aws/eks-anywhere/eks-anywhere-packages-bundles"
+	EksaPackagesNamespace              = "eksa-packages"
+	EksaPackageControllerHelmChartName = "eks-anywhere-packages"
+	EksaPackagesSourceRegistry         = "public.ecr.aws/l0g8r8j6"
+	EksaPackageControllerHelmURI       = "oci://" + EksaPackagesSourceRegistry + "/eks-anywhere-packages"
+	EksaPackageControllerHelmVersion   = "0.1.14-eks-a-v0.0.0-dev-build.3624"
+	EksaPackageBundleURI               = "oci://" + EksaPackagesSourceRegistry + "/eks-anywhere-packages-bundles"
 )
 
-var (
-	eksaPackageControllerHelmValues = []string{}
-	eksAnywherePackagesHelmValues   = []string{"sourceRegistry=public.ecr.aws/l0g8r8j6"}
-)
+var EksaPackageControllerHelmValues = []string{"sourceRegistry=public.ecr.aws/l0g8r8j6"}
+
+type resourcePredicate func(string, error) bool
+
+func NoErrorPredicate(_ string, err error) bool {
+	return err == nil
+}
+
+// TODO turn them into generics using comparable once 1.18 is allowed
+func StringMatchPredicate(s string) resourcePredicate {
+	return func(in string, err error) bool {
+		return err == nil && strings.Compare(s, in) == 0
+	}
+}
+
+func IntEqualPredicate(i int) resourcePredicate {
+	return func(in string, err error) bool {
+		inInt, err := strconv.Atoi(in)
+		return err == nil && inInt == i
+	}
+}
+
+func WaitForResource(
+	test *framework.ClusterE2ETest,
+	ctx context.Context,
+	resource string,
+	namespace string,
+	jsonpath string,
+	timeout time.Duration,
+	predicates ...resourcePredicate,
+) error {
+	end := time.Now().Add(timeout)
+	if !strings.HasPrefix(jsonpath, "jsonpath") {
+		jsonpath = fmt.Sprintf("jsonpath='%s'", jsonpath)
+	}
+	command := []string{}
+	for time.Now().Before(end) {
+		out, err := test.KubectlClient.Execute(ctx, "get", "-n", namespace,
+			"--kubeconfig="+test.Cluster().KubeconfigFile,
```

Comment:
```suggestion
			"--kubeconfig="+kubeconfig.FromClusterName(test.ClusterName),
```
This way you don't have to export Cluster().